### PR TITLE
Additional Bugfixing

### LINKED ISSUE #1641: BUG: P E N G is literally stronger than bullets
#### Description of issue:
if you throw P E N G at someone, they will take so much damage, that their limbs can literally instantly explode, in a single P E N G throw

#### Difference between expected and actual behavior:
throwing a P E N G at someone and they get minor wounds

throwing a P E N G at someone and one of their limbs explodes
#### Steps to reproduce:
throw P E N G at someone

### LINKED ISSUE #1826: Bug : adding salt to empty condiment bottles causes them to show up as the missing texture box
#### Steps to reproduce: grab bag of salt from freezer, add salt to empty condiment bottle

F99F9EC5-C4D0-45E4-B321-6A087C749746

### LINKED ISSUE #2004: Suggestion: buff CE base stats
#### Suggestion:
Buff the CE's construction skill from experienced to master
Buff the CE's electrics skill from experienced to master
buff the CE's EVA skill from trained to experienced
#### What do you think it'd add:
The CE is a highly trained engineer, it just makes sense for him to be very good at using his RIG and making shit.

### LINKED ISSUE #2014: BUG:shitcurity boots can't be slipped
#### Description of issue:
/obj/item/clothing/shoes/pcsi has slip immunity

#### Difference between expected and actual behavior:
Soap is soap, you slip on it, you fall, be robust or get slipped

seccies can't focken slip

### LINKED ISSUE #2038: Suggestion: Remove Yellow Glow from Scry

### PR DESCRIPTION
**Pull request by JakeDaBoss**
Because 25 is not even what the fire axe has.

Closes #1641
Closes  #1826
Closes  #2004
Closes  #2014
Closes  #2038

### CODE REVIEW

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 1]
```diff
@@ -156,7 +156,6 @@
 	icon_state = "combat"
 	force = 3
 	armor = list(melee = 80, bullet = 60, laser = 50, energy = 40, bomb = 50, bio = 10, rad = 0)
-	item_flags = ITEM_FLAG_NOSLIP
```
Comment (DTraitor) [inline code/modules/clothing/shoes/miscellaneous.dm:159]: This is a feature on many server. West is ok with `item_flags = ITEM_FLAG_NOSLIP` as long as it is intended

File: code/modules/economy/peng.dm [hunk 2]
```diff
@@ -18,8 +18,8 @@ GLOBAL_DATUM(peng, /obj/item/weapon/peng)
 	icon = 'icons/obj/economy.dmi'
 
 	var/claimed = FALSE
-	force = 25	//Its pretty good to whack people with
-	throwforce = 25
+	force = 6	//Its pretty good to whack people with - Not anymore.
```
Comment (DTraitor) [inline code/modules/economy/peng.dm]: I think you can remove this comment

**Review by DTraitor — CHANGES_REQUESTED** [review]
Just some minor tweaks, don't forget to update changelog

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 1 above)
Comment (JakeDaBoss) [reply code/modules/clothing/shoes/miscellaneous.dm:159]: #2014 will need to be closed then, as this was tagged feature adjustment.

**Pull request closed without merging**